packages/data/single-file/src/is-typedarray.ts

```diff
@@ -0,0 +1,5 @@
+import whichTypedArray from '@nolyfill/which-typed-array';
+
+export default function isTypedArray(value: unknown) {
+  return !!whichTypedArray(value);
+}
```

Comment:
@asukaminato0721 

https://unpkg.com/is-typedarray@1.0.0/index.js

The `is-typedarray` implementation is actually faster and more lightweight even than `@nolyfill/which-typed-array`. We should not replace that.